<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManagerForm</class>
 <widget class="QWidget" name="OrderManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="orderInfoLabel">
           <property name="font">
            <font>
             <family>휴먼매직체</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>OrderManagerInfo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="clientComboBox">
             <property name="minimumSize">
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ClientID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Name</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Address</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="clientLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="clientButton">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="productComboBox">
             <property name="minimumSize">
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ProductID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Name</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="productLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="productButton">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeView" name="clientTreeView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="productTreeView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="OrderformLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>&amp;OrderNum</string>
           </property>
           <property name="buddy">
            <cstring>idLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="idLineEdit">
           <property name="text">
            <string>Auto-generated</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="cIdLabel">
           <property name="text">
            <string>&amp;Client</string>
           </property>
           <property name="buddy">
            <cstring>clientLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="cIdLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="pIdLabel">
           <property name="text">
            <string>&amp;Product</string>
           </property>
           <property name="buddy">
            <cstring>productLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="pIdLineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="stockLabel">
           <property name="text">
            <string>&amp;Stock</string>
           </property>
           <property name="buddy">
            <cstring>stockLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="stockLineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Only Number</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="addressLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="priceLabel">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="priceLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="sumLabel">
           <property name="text">
            <string>Sum</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="sumLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addPushButton">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifyPushButton">
             <property name="text">
              <string>&amp;Modify</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>&amp;Search</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>OrderNum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Client (Name or ID)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Product (Name or ID)</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="searchPushButton">
              <property name="text">
               <string>S&amp;earch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="statePushButton">
              <property name="text">
               <string>All Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="treewidgetLayout" stretch="0">
       <item>
        <widget class="QTableView" name="orderTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientLineEdit</tabstop>
  <tabstop>clientButton</tabstop>
  <tabstop>productLineEdit</tabstop>
  <tabstop>productButton</tabstop>
  <tabstop>clientTreeView</tabstop>
  <tabstop>productTreeView</tabstop>
  <tabstop>idLineEdit</tabstop>
  <tabstop>cIdLineEdit</tabstop>
  <tabstop>pIdLineEdit</tabstop>
  <tabstop>stockLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>priceLineEdit</tabstop>
  <tabstop>sumLineEdit</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>modifyPushButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>searchComboBox</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
  <tabstop>statePushButton</tabstop>
  <tabstop>orderTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
